pyrokinetics/gk_code/GKInputCGYRO.py

```diff
@@ -184,8 +233,15 @@ def get_local_geometry(self) -> LocalGeometry:
         Returns local geometry. Delegates to more specific functions
         """
         eq_type = self.cgyro_eq_types[self.data["EQUILIBRIUM_MODEL"]]
+
+        is_basic_miller = self._check_basic_miller()
+        if eq_type == "MXH" and is_basic_miller:
+            eq_type = "Miller"
```

Comment:
Warn to the user that the type is being reduced (i.e. information lost)?

Reply:
Its not that the information is being lost, but that the squareness and higher moments are all 0 so it doesn't exist to begin with. Essentially, CGYRO without those values is the 1998 Miller so we re-define it as such.